vector/src/main/java/im/vector/app/features/crypto/verification/self/SelfVerificationViewModel.kt

```diff
@@ -204,8 +204,18 @@ class SelfVerificationViewModel @AssistedInject constructor(
                 }
             }
             is VerificationAction.GotItConclusion -> {
-                // just dismiss
-                _viewEvents.post(VerificationBottomSheetViewEvents.Dismiss)
+                withState { state ->
```

Comment:
I think there is more to do for that use case.
You would have to intercept back (swipe) navigation events. For a bottom sheet probably by overriding onCreateDialog() and add a KeyListener (should be in the old VerificationBottomSheet)

Then when intercept you delegate to view model (queryCancel()) that depending on the state would do different things.
E.g if there is an outgoing request you might want to prompt for cancel, eventually cancel and go back to main screen where you can use 4S/Reset etc..